### PR TITLE
0.8 port

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

File: google-calendar.html [hunk 1]
```diff
@@ -46,42 +44,64 @@
...
+      on-google-api-load="displayCalendars"></google-client-loader>
+    <google-signin-aware
+      scopes="https://www.googleapis.com/auth/calendar.readonly"
+      on-google-signin-aware-success="signIn"
+      on-google-signin-aware-signed-out="signOut"></google-signin-aware>
+
+    <ul id="calendars">
+      <li>{{title}}</li>
+      <template is="x-repeat" items="{{calendars}}">
+        <li style$="{{ computeCalStyle(item.backgroundColor)}}">
```
Comment (ebidel) [inline google-calendar.html]: `{{computed...`

File: google-calendar.html [hunk 2]
```diff
@@ -46,42 +44,64 @@
...
+       * @attribute title
+       * @type string
+       */
+      title: {
+        type: String,
+        value: 'My calendars'
+      },
+      calendars: {
+        type: Array,
+        value: function() { return []}
```
Comment (ebidel) [inline google-calendar.html]: `{ return []; }

File: google-calendar.html [hunk 3]
```diff
@@ -91,13 +111,17 @@
       if (this.signedIn && this.$.calendar.api) {
         var request = this.$.calendar.api.calendarList.list({"key": ""});
         request.execute(function(resp) {
-          this.calendars = resp.items;
+          if (resp.error) {
+            console.error("Error with calendarList.list", resp.message)
+          }
+          else {
```
Comment (ebidel) [inline google-calendar.html]: else should be on the same line as `}`

File: google-calendar.html [hunk 4]
```diff
@@ -117,124 +141,142 @@
...
+        },
         /**
          * Label to be displayed if the status is busy.
          * @attribute busyLabel
          * @type string
          */
-        busyLabel: 'I\'m busy',
+        busyLabel: {
+          type: String,
+          value: 'I\'m busy'
```
Comment (ebidel) [inline google-calendar.html]: "I'm busy"

File: google-calendar.html [hunk 5]
```diff
@@ -117,124 +141,142 @@
...
          * @type string
          */
-        freeLabel: 'I\'m free',
-        className: '',
-        label: '',
-        calendarIdChanged: function() {
-          this.displayBusy();
+        freeLabel: {
+          type: String,
+          value: 'I\'m free'
```
Comment (ebidel) [inline google-calendar.html]: same

File: google-calendar.html [hunk 6]
```diff
@@ -117,124 +141,142 @@
...
-          * Displays the busy/free status.
-          * @method displayBusy
-          */
-        displayBusy: function() {
-          if (!this.calendarId) {
-            console.log('CalendarId is required for this component');
-            return;
+      },
+
+      className: '',
```
Comment (ebidel) [inline google-calendar.html]: Are these private? They should have `_` prefixes

File: google-calendar.html [hunk 7]
```diff
@@ -117,124 +141,142 @@
...
-          if (this.$.calendar.api) {
-            if (this.apiKey) {
-              gapi.client.setApiKey(this.apiKey);
+          var now = new Date();
+          var query = {
+            timeMin: now.toISOString(),
+            timeMax: new Date(now.valueOf() +
+                TIME_SPAN * MS_PER_MINUTE).toISOString(),
+            items: [
+              {
```
Comment (ebidel) [inline google-calendar.html]: I know you didn't do this, but the spacing here is really funky.

File: demo.html [hunk 8]
```diff
@@ -18,10 +18,10 @@
   </style>
 
 </head>
```
Comment (ebidel) [inline demo.html]: I think 0.8 demos should be in `demos/index.html`. Is that still the case?

File: google-calendar.html (hunk 1 above)
Comment (atotic) [inline google-calendar.html]: fixed.

File: google-calendar.html (hunk 2 above)
Comment (atotic) [inline google-calendar.html]: fixed.

File: google-calendar.html (hunk 3 above)
Comment (atotic) [inline google-calendar.html]: fixed

File: google-calendar.html (hunk 6 above)
Comment (atotic) [inline google-calendar.html]: We'll have to do a whole lot of prefixing to make our docs look good later. All private methods will need _. Delayed for the big doc powwow.

File: google-calendar.html (hunk 7 above)
Comment (atotic) [inline google-calendar.html]: fixed.

File: demo.html (hunk 8 above)
Comment (atotic) [inline demo.html]: I think that's a should, not a must. iron-doc defaults to demo/index.html, but you can override. I left them in place for now, can do wholesale move in doc migration.

**Comment by atotic** [comment]
fixes checked in.

File: demo.html (hunk 8 above)
Comment (ebidel) [inline demo.html]: k

File: google-calendar.html (hunk 6 above)
Comment (ebidel) [inline google-calendar.html]: Based on slack, I think we should move these two properties into `properties` so data binding picks them up

Comment (atotic) [inline google-calendar.html]: done

File: demo.html (hunk 8 above)
Comment (addyosmani) [inline demo.html]: @ebidel fwiw, most of the current PRs for GWC ports continue to use demo.html. We might want to change that to demo/index.html on the next iteration through.

**Comment by ebidel** [comment]
LGTM

**Pull request merged**